### PR TITLE
fix(github): change pr title type to long text

### LINKED ISSUE #2363: [Bug][Github] data too long for column 'title'
### Search before asking

- [X] I had searched in the [issues](https://github.com/apache/incubator-devlake/issues?q=is%3Aissue) and found no similar issues.

### What happened

when importing https://github.com/BVLC/caffe to devlake v0.11, the following error occurs:

image

### What you expected to happen

Apparently the "title" column in some recipient table (PR? issues?) is defined too short.

### How to reproduce

just use the Github plugin to import https://github.com/BVLC/caffe 

```json
[
  [
    {
      "Plugin": "github",
      "Options": {
        "repo": "caffe",
        "owner": "BVLC"
      }
    }
  ]
]
```

### Anything else

_No response_

### Version

v0.11.0

### Are you willing to submit PR?

- [ ] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

### PR DESCRIPTION
**Pull request by warren830**
# Summary

change github.pr.title from varchar(255) to longtext

### Does this close any open issues?
closes #2363 

### Screenshots
image

### Other Information
Any other information that is important to this PR.

### CODE REVIEW

**Comment by likyh** [comment]
Maybe you can target to add some e2e tests.

File: plugins/github/impl/impl.go [hunk 1]
```diff
@@ -55,24 +55,24 @@ func (plugin Github) SubTaskMetas() []core.SubTaskMeta {
...
-		tasks.CollectApiPullRequestCommitsMeta,
-		tasks.ExtractApiPullRequestCommitsMeta,
-		tasks.CollectApiPullRequestReviewsMeta,
-		tasks.ExtractApiPullRequestReviewersMeta,
-		tasks.CollectApiCommitsMeta,
-		tasks.ExtractApiCommitsMeta,
-		tasks.CollectApiCommitStatsMeta,
-		tasks.ExtractApiCommitStatsMeta,
-		tasks.EnrichPullRequestIssuesMeta,
+		//tasks.CollectApiCommentsMeta,
```
Comment (likyh) [inline plugins/github/impl/impl.go:58]: Emmm

Comment (warren830) [reply plugins/github/impl/impl.go:58]: my bad...

**Review by likyh — APPROVED** [review]
LGTM

**Review by abeizn — APPROVED** [review]
LGTM

**Pull request merged**